README.adoc

```diff
@@ -5,3 +5,12 @@ NOTE: Extends link:https://github.com/jboss-container-images/jboss-openjdk-image
 # License
 
 See link:LICENSE[LICENSE] file.
+
+# TODO
+
+- [ ] Enable GitHub issue to allow people to give feedback
+- [ ] Add instructions how to recreate the image
+- [ ] Add some background information what build system is used and how it works
+- [ ] Add some information about the image features
```

Comment:
We need to understand first what is the best place for this information. We're currently handing over these images to the product team. This image is a bit special and needs discussion. We need to maintain the same workflow for all images, so I would like to first agree how to do it and where.